tests/v2/utils/library/test_utils.go

```diff
@@ -281,13 +293,43 @@ func verify(devfile *commonUtils.TestDevfile) error {
 		if components != nil && len(components) > 0 {
 			err := VerifyComponents(devfile, components)
 			if err != nil {
-				errorString = append(errorString, commonUtils.LogErrorMessage(fmt.Sprintf("Verfify Components %s : %v", devfile.FileName, err)))
+				errorString = append(errorString, commonUtils.LogErrorMessage(fmt.Sprintf("Verify Components %s : %v", devfile.FileName, err)))
 			}
 		} else {
 			commonUtils.LogInfoMessage(fmt.Sprintf("No components found in %s : ", devfile.FileName))
 		}
 	}
 
+	commonUtils.LogInfoMessage(fmt.Sprintf("Get projects %s : ", devfile.FileName))
+	projects, err := libraryData.GetProjects(common.DevfileOptions{})
+	if err != nil {
+		errorString = append(errorString, commonUtils.LogErrorMessage(fmt.Sprintf("Getting Projects from library : %s : %v", devfile.FileName, err)))
+	} else {
+		if projects != nil && len(projects) > 0 {
+			err := VerifyProjects(devfile, projects)
+			if err != nil {
+				errorString = append(errorString, commonUtils.LogErrorMessage(fmt.Sprintf("Verify Projects %s : %v", devfile.FileName, err)))
+			}
+		} else {
+			commonUtils.LogInfoMessage(fmt.Sprintf("No projects found in %s : ", devfile.FileName))
+		}
+	}
+
+	commonUtils.LogInfoMessage(fmt.Sprintf("Get projects %s : ", devfile.FileName))
```

Comment:
mention starter projects, in the future this may be a source of confusion when debugging!

Reply:
Thx, fixed.